JS/Lesson15/saffet/script.js

```diff
@@ -67,113 +67,348 @@ switch (day) {
-checkIfStringIsUpperCase('write a program that takes in');
-
+checkUpperCase("Saffet");
+checkUpperCase("ramazan");
+console.log("-------------------------------------------------");
 // 5. Write a program that takes in a number and checks if it's positive, negative, or zero using if else statement.
+console.log("Ex5:");
 
-// 6. Write a program that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+function checkSign(number){
+	if(number < 0){
+		console.log("The number is negative (-)");
+	}
+	else if(number > 0){
+		console.log("The number is positive (+)");
+	}
+	else{
+		console.log("The number is zero (0)");
+	}
+}
+checkSign(3);
+checkSign(-4);
+checkSign(0);
+
+console.log("-------------------------------------------------");
+// 6. Write a program that takes in a year and checks if it's a leap year 
+//(is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+console.log("Ex6:");
+
+function checkLeapYear(year){
+	if(year % 4 == 0 && year % 400 == 0){
+		console.log("It's leap year");
+	}
+	else{
+		console.log("It is not leap year");
+	}
+}
+
+checkLeapYear(1800); // it is not leap year.
+checkLeapYear(1600); // it is a leap year.
 
+console.log("-------------------------------------------------");
 // 7. Write a program that takes in a temperature and checks if it's above or below freezing using if else statement.
+console.log("Ex7:");
 
+function checkTemp(temperature){
+	if(temperature <= 0){
+		console.log("Below freezing")
+	}
+	else{
+		console.log("Above freezing");
+	}
+}
+checkTemp(-10);
+checkTemp(10);
+checkTemp(0);
+
+console.log("-------------------------------------------------");
 // 8. Write a program that takes in a person's height and weight and checks if they are considered underweight, normal, overweight, or obese using if else statement.
 // Body Mass Index (BMI) is a person’s weight in kilograms (or pounds) divided by the square of height in meters (or feet). A high BMI can indicate high body fatness.
 //Formula: weight (kg) / [height (m)]2
 // Below 18.5	Underweight
 // 18.5 – 24.9	Healthy Weight
 // 25.0 – 29.9	Overweight
 // 30.0 and Above	Obesity
+console.log("Ex8:");
+function chechBody(weight, height){
+	let bmi = weight / (height*height);
+
+	if (bmi < 18.5)
+	console.log("underweight");
 
+	else if (bmi >= 18.5 && bmi < 24.9)
+	console.log("Healthy");
+
+	else if (bmi >= 24.9 && bmi < 30)
+   console.log("overweight");
+
+	else if (bmi >= 30)
+	console.log("Obesity");
+}
+
+chechBody(25, 1.69);
+chechBody(75, 1.69);
+chechBody(100, 1.69);
+
+
+console.log("-------------------------------------------------");
 // 9. Write a program that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
+console.log("Ex9:");
+function checkCategory(age1){
```

Comment:
You should not have to write age1 as a name of a function parameter. The parameter is local to the scope of the function so as long as there are no other parameter with name "age" that this function takes (and it does not) and no other variables with name "age" declared inside the function itself, it should be safe to name the parameter "age" instead of  "age1"

Reply:
I understand, just I thought that there are different functions and I need to seperate these functions' parameters easily. Thanks